src/hotspot/os/aix/os_aix.cpp

```diff
@@ -96,6 +96,10 @@
 #include <sys/ioctl.h>
 #include <sys/ipc.h>
 #include <sys/mman.h>
+// sys/mman.h defines MAP_ANON_64K beginning with AIX7.3 TL1
+#ifndef MAP_ANON_64K
+#define MAP_ANON_64K  0x400
+#endif
```

Comment:
```suggestion
// sys/mman.h defines MAP_ANON_64K beginning with AIX7.3 TL1
#ifndef MAP_ANON_64K
#define MAP_ANON_64K  0x400
#else
STATIC_ASSERT(MAP_ANON_64K == 0x400);
#endif
```
That adds a test, for when you *do* have the MAP_ANON_64K constant, that your assumption about its value (0x400) is correct. Needs utilities/debug.hpp, but you should include that anyway.